src/main/resources/align-confluent-kafka.json

```diff
@@ -0,0 +1,18 @@
+{
+    "align": [
+        {
+            "name": "align-archaius2",
```

Comment:
Hi @Fleshgrinder 

Thanks for the contribution.

Any chance you could rename this rule to `align-confluent-kafka` or similar? otherwise this would replace existing rules

Also, sorry for the late response :( totally missed your PR

Reply:
That was an oversight on my part, of course we have to rename it. 😉

> Also, sorry for the late response :( totally missed your PR

No worries, better late than never. I developed the habit of closing open source PRs after a couple of months of no response because there simply are too many projects where contributions are totally ignored.